### ISSUE TITLE
.nav-tabs and .nav-pills ... should they always be role="tablist"?

### ISSUE THREAD

**Issue by patrickhlauke** [paypal/bootstrap-accessibility-plugin#59]
This is in the context of https://github.com/patrickhlauke/bootstrap/commit/3425f961eed0f74695ca5f92f8e2bc1017febc4c#commitcomment-8379233 where I propose to drop the various role="tablist" from the "Navs" code as, in most cases, I'd argue those .nav-tabs and .nav-pills won't be used for actual tab panels, but to quickly style up a mini navigation (in which case role="tablist" would be wrong).

However, your tab.js is quoted as a counter-example https://github.com/paypal/bootstrap-accessibility-plugin/blob/ecf76994563d6e73f01a8ecb78949dc2993368a3/src/js/tab.js#L8 ... anything with .nav-tabs and .nav-pills automatically gets the role="tablist"

i do call out the fact that if it is in fact used for a tablist, it should have the additional roles in line 24 https://github.com/patrickhlauke/bootstrap/commit/3425f961eed0f74695ca5f92f8e2bc1017febc4c#diff-4807a8cd0881fb6cce944561f89bce81L24

In short: any thoughts on this? If people do use the "Navs" for something other than real tab panels (which I would argue they would), should it still be given role="tablist" which seems incorrect?
